Assets/Scripts/ExplosionParticleController.cs

```diff
@@ -0,0 +1,38 @@
+using UnityEngine;
+
+namespace DefaultNamespace
```

Comment:
why is this in a namespace?

Reply:
I'm actually in favor of splitting classes into folders and namespaces, but Rider auto-adds it to `DefaultNamespace` when creating a class with no obvious folder structure. It looks to be removed in another revision.